mock/py/mock.py

```diff
@@ -556,11 +559,11 @@ def check_arch_combination(target_arch, config_opts):
             time.sleep(5)
 
 @traceLog()
-def do_debugconfig(config_opts, uidManager):
+def do_debugconfig(config_opts, uidManager, expand = False):
```

Comment:
nit: can you drop the spaces around `=` sign?

Reply:
done